client/libs/theme/_fonts.scss

```diff
@@ -17,8 +17,11 @@
 
 // Text Color
 .primary-text { color: $primary !important; }
+.bg-primary { background-color: $primary !important; }
 .accent-text { color: $accent !important; }
+.accent-icon { color: $accent !important; }
 .secondary-text { color: $secondary !important; }
+.secondary-icon { color: $secondary !important; }
```

Comment:
Wherever you used `secondary-icon` replace that with `secondary-text` since the material icons are technically text.
```suggestion
```